src/stores/useSubmitStore.js

```diff
@@ -0,0 +1,30 @@
+import { create } from "zustand";
+
+
+const useSubmitStore = create((set) => ({
```

Comment:
This store is only used in one component, which makes it unnecessary and we can use local state instead